doc/user_guide/migrating.rst

```diff
@@ -0,0 +1,284 @@
+Migrating Legacy Projects
+=========================
+
+Migrating old projects to a new project setup and the :code:`python-toolbox` can be tedious and annoying. This guide will try to provide some basic steps and guidance to simplify this process.
+
+.. _before_you_migrate:
+
+What a Project Should Have Before You Migrate
++++++++++++++++++++++++++++++++++++++++++++++
+
+* The project configuration should be :code:`pyproject.toml` based. **[Required]**
+* The project should be :code:`poetry` based. **[Required]**
+* Dependencies should point only to officially published dependencies on PyPI (no git references or similar) **[Required]**.
+* The project documentation should be :code:`sphinx` based **[Required]**.
+* Automated tasks within the project should be available as Python code or `Nox`_ tasks. **[Helpful]**
```

Comment:
Maybe also mention the expected test folder structure?
test/unit test/integration?

Reply:
I think a quick look in the standard structure of a new project as mentioned bellow is better as it is more likely to stay in sync with the project needs.